internal/consensus/pbts_test.go

```diff
@@ -71,13 +73,19 @@ type pbtsTestConfiguration struct {
 
 	// The timestamp of the block proposed at height 2.
 	height2ProposedBlockTime time.Time
+
+	// The timestamp of the block proposed at height 4.
+	height4ProposedBlockTime time.Time
 }
 
 func newPBTSTestHarness(ctx context.Context, t *testing.T, tc pbtsTestConfiguration) pbtsTestHarness {
 	t.Helper()
 	const validators = 4
 	cfg := configSetup(t)
 	clock := new(tmtimemocks.Source)
+	if tc.height4ProposedBlockTime.IsZero() {
+		tc.height4ProposedBlockTime = tc.height2ProposalDeliverTime.Add(time.Millisecond * 2)
```

Comment:
Is 2ms here just "a while later"?

If so that's fine, though it might be worth having a comment to let the reader know what this needs to be longer than.

Reply:
Added several comments and a `blockTimeIota` to hopefully clarify.